lib/src/serializable.dart

```diff
@@ -69,8 +72,11 @@ class _ModuleRegistration extends Object with Disposable {
   _ModuleRegistration(this.module);
 
   @override
-  void manageStreamSubscription(StreamSubscription subscription) {
-    super.manageStreamSubscription(subscription);
+  StreamSubscription<T> listenToStream<T>(
+      Stream<T> stream, void onData(T event),
+      {Function onError, void onDone(), bool cancelOnError}) {
+    return super.listenToStream(stream, onData,
+        onError: onError, onDone: onDone, cancelOnError: cancelOnError);
```

Comment:
this seems to just be proxying directly to the super class with no changes – is an override actually needed?